src/data/player.py

```diff
@@ -283,16 +287,12 @@ def rating_type(self) -> PlayerRatingType:
     def point_values(self) -> dict[Result, float] | None:
         return self.tournament.point_values if self.tournament else None
 
-    def compute_points(self, max_round: int):
-        """Computes and stores the points of the player,
-        from round 1 to round `max_round` (returns None)"""
-        # NOTE(Amaras) this does not rely on the fact that insertion order
-        # is preserved in 3.6+ dict, because I can't be sure insertion order
-        # is the correct (increasing) round order
-        # NOTE(Amaras) if you were to include the current round
-        # in the computation, boards regularly change their ordering
-        # during the current round as results are added
-        self.points = self.points_before(max_round)
+    def compute_points(
+            self,
+            before_round: int
+    ):
```

Comment:
maybe this should be a keyword argument to avoid ambiguity
```suggestion
    def compute_points(
            self,
            *,
            before_round: int
    ):
```